person.rb

```diff
@@ -0,0 +1,38 @@
+class Person
+  def initialize(id, age, name = 'Unknown', parent_permission: true)
+    @id = id
+    @age = age
+    @name = name
+    @parent_permission = parent_permission
+  end
+
+  def getid
+    @id
+  end
+
+  def getage
+    @age
+  end
+
+  def getname
+    @name
+  end
+
+  def setname=(name)
+    @name = name
+  end
+
+  def setage=(age)
+    @age = age
+  end
```

Comment:
- [ OPTIONAL] Instead of writing multiple methods just to get attributes and set them you easily use `attr_reader which will allow you to read the attribute and attr_accessor it will allow you to read and set the attributes` which will do the same thing but in one line of code as the following g
```suggestion
  attr_accessor :name, :age 
  attr_reader :id 

```